tests/qat/test_config.py

```diff
@@ -105,6 +107,24 @@ def test_config_repeats(self):
         assert first_conf.repeats == second_conf.repeats
         assert first_conf.repetition_period == second_conf.repetition_period
 
+    @pytest.mark.parametrize(
+        "repeats, possible_error_throw",
+        [
+            (64_000, does_not_raise()),
+            (100_000, does_not_raise()),
+            (100_001, pytest.raises(ValueError, match="exceeds the maximum amount of")),
+            (1_000_000, pytest.raises(ValueError, match="exceeds the maximum amount of")),
+            (0, does_not_raise()),
```

Comment:
Super minor.
Could these be ordered `None` - highest?
Also, suggest using [PyTest ids](https://docs.pytest.org/en/stable/example/parametrize.html#different-options-for-test-ids) to make the output more informative.

Some examples: 
* https://github.com/oqc-community/qat/blob/7df92aed503ae4a7f6a11ec22965c3fa831a9b99/tests/qat/test_tket.py#L22
* https://github.com/oqc-community/qat/blob/7df92aed503ae4a7f6a11ec22965c3fa831a9b99/tests/qat/test_instructions.py#L264
* https://github.com/oqc-community/qat/blob/7df92aed503ae4a7f6a11ec22965c3fa831a9b99/tests/qat/test_instructions.py#L312